code/modules/client/preference_setup/general/03_body.dm

```diff
@@ -373,9 +373,7 @@ var/global/list/valid_bloodtypes = list("A+", "A-", "B+", "B-", "AB+", "AB-", "O
 		var/status = pref.organ_data[name]
 		var/obj/item/organ/external/O = character.organs_by_name[name]
 		if(O)
-			if(status == null)
-				O.derobotize()

```

Comment:
Keeping the derobotize proc, but reintroducing the memory leak means we don't worry about this.